eventing/README.md

```diff
@@ -230,6 +230,9 @@ FTP server for new files or generate events at a set time interval.
 - [Install Eventing components](#installation)
 - [Run samples](samples/)
 
+## Configuration
+- [Default Channels](default-channels.md)
```

Comment:
Add a one-sentence explanation of what default channels are here, so people don't have to click the link to know if they want to read more? e.g.

```suggestion
- [Default Channels](default-channels.md) provide a way to choose the persistence strategy for Channels across the cluster.
```